presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PropertyDerivations.java

```diff
@@ -286,8 +292,8 @@ public ActualProperties visitTopNRowNumber(TopNRowNumberNode node, List<ActualPr
 
             ImmutableList.Builder<LocalProperty<Symbol>> localProperties = ImmutableList.builder();
             localProperties.add(new GroupingProperty<>(node.getPartitionBy()));
-            for (Symbol column : node.getOrderBy()) {
-                localProperties.add(new SortingProperty<>(column, node.getOrderings().get(column)));
+            for (Symbol column : node.getOrderingScheme().getOrderBy()) {
```

Comment:
wrong commit

Reply:
ditto, `WindowNode.Specification` changes propagate to other nodes.